lookout/style/format/langs/__main__.py

```diff
@@ -155,22 +172,59 @@ def analyze_file(path: str):
                 log.warning("dropped %s - language mismatch %s != %s",
                             path, language, response.language)
                 return
-            analyze_uast(path, response.uast, roles, reserved)
+            content = Path(path).read_text()
+            analyze_uast(path, content, response.uast, roles, uast_roles, reserved)
             progress.update(1)
         except:  # noqa: E722
             log.exception("Parsing %s", path)
             errors = True
 
-    with progress:
-        for file in args.input:
-            pool.submit(analyze_file, file)
+    def analyze_parquet_row(row: pd.Series, filepath):
+        nonlocal errors
+        if errors:
+            return
+        nonlocal language
+        if not language:
+            log.warning("language must be specified for parquet files handling")
+            errors = True
+            return
+        try:
+            path = "%s:%s" % (filepath, row.path)
+            analyze_uast(path, row.content.decode(), bblfsh.Node.FromString(row.uast),
```

Comment:
decode with utf-8 errors ignored please - and I am not talking about `except UnicodeDecodeError`